js/modules/loki_snode_api.js

```diff
@@ -84,6 +78,31 @@ class LokiSnodeAPI {
     }
   }
 
+  async getSwarmNodesForPubKey(pubKey) {
+    let swarmNodes;
+    try {
+      const conversation = ConversationController.get(pubKey);
+      swarmNodes = [...conversation.get('swarmNodes')];
```

Comment:
```suggestion
      const swarmNodes = [...conversation.get('swarmNodes')];
      return swarmNodes;
```
You can just do that